playbooks/roles/airship-setup-deployer/templates/helm-serve.service.j2

```diff
@@ -3,7 +3,7 @@ Description=Helm Server
 After=network.target
 
 [Service]
-User=root
+User={{ ansible_user }}
```

Comment:
Could you clarify how do you intent to use this?

This basically expects the ssh plugin to be setting ansible_user. So you need to gather facts and use said plugin.

There is also a need to understand if we're gonna use become for this or not, and where the user comes from (localhost or the deployer node). According to the commit message, I would say this is the deployer node, so I assume this ansible_user is set in the inventory for ssh connection plugin usage.